### DISCUSSION TITLE
Custom folders

### DISCUSSION THREAD

**Discussion by SupaStar** [opcodesio/log-viewer#44] [Ideas]
Hello I just installed this package in my project, but I have a small problem, I use folders to define the type of log that is generated, is it possible that these same ones are detected?
Example
logs/error
logs/info
logs/etc...
![2](https://user-images.githubusercontent.com/68682799/187824462-b0711e99-35a3-4f5f-a3b0-eab3b25f6652.png)
![1](https://user-images.githubusercontent.com/68682799/187824464-57151d71-7005-46ab-bc21-7383790fc150.png)

**Comment by arukompas** [comment]
@SupaStar hey!

First of all, make sure you update the `include_files` attribute in `config/log-viewer.php` to include your subfolders as well, and then it will be able to read those log files.

1. Folder listing is a feature that's coming later 😄 so just start the repository and keep an eye on it

2. With Log Viewer's filters available for the different log severities (info, debug, warning, error, etc), do you think you still need to split your log files into different folders? You could easily filter what severities you want to see directly in the Log Viewer and it can handle large log files. Just an idea - you might want to experiment with that in the meantime :)

**Reply by vesper8** [reply]
Please allow us to specify absolute paths to folders in the `include_files`, I had to use a lot of `../../` to get to the folders I wanted to read from. And as a result of this, the names of the files in the left column are now also showing a bunch of `../../`.

I would prefer to see only the name of the file there and not the full path, relative or absolute as it may be.

**Reply by arukompas** [reply]
@vesper8 I heard your prayers! Please update to the latest version, which has an updated folder-friendly UI and allows for absolute paths as well.

**Reply by vesper8** [reply]
Thank you!

I have some more prayers too.. I'll make a new post for these : )
